ExposureNotification.App/ExposureNotification.App/ViewModels/WelcomeViewModel.cs

```diff
@@ -12,15 +12,15 @@ public class WelcomeViewModel : ViewModelBase
 	{
 		public WelcomeViewModel()
 		{
-			Xamarin.ExposureNotifications.ExposureNotification.IsEnabledAsync()
-				.ContinueWith(async t =>
-				{
-					if (t.Result)
-					{
-						IsEnabled = true;
-						await GoToAsync($"//{nameof(InfoPage)}");
-					}
-				});
+			//Xamarin.ExposureNotifications.ExposureNotification.IsEnabledAsync()
```

Comment:
meant to be commented out?

Reply:
Nope, accidentally committed that.. fixed.